packages/dappmanager/src/domains.ts

```diff
@@ -1,53 +1,75 @@
 import { params } from "@dappnode/params";
 import { getContainerDomain } from "@dappnode/types";
 
-export function stripCharacters(s: string): string {
-  return s.replace(RegExp("_", "g"), "");
-}
-
-export function stripBadDomainChars(s: string): string {
-  // eslint-disable-next-line no-useless-escape
-  return s.replace(/[^a-zA-Z\-]+/g, "-");
-}
+export type ContainerNames = { serviceName: string; dnpName: string };
 
 /**
- * - Strip container prefix
- * - Strip .dappnode, .eth, .dnp
- * - Strip "_"
+ * This function return fullNetworkAlias or shortNetworkAlias:
+ * - shortDnpName is dnpName.split(0)["."]. i.e dnpName = "dappmanager.dnp.dappnode.eth" => shortDnpName = "dappmanager"
  *
- * @param name "bitcoin.dnp.dappnode.eth"
- * @returns "bitcoin"
- * - "bitcoin.dnp.dappnode.eth" > "bitcoin"
- * - "other.public.dappnode.eth" > "other.public"
+ * - i.e fullNetworkAlias: (serviceName..dappnode) "dappmanager.dnp.dappnode.eth.dappmanager.dappnode"
+ * - i.e shortNetworkAlias: () "dappmanager.dappnode"
+ *
+ * - For dnp packages ( example.dnp.dappnode.eth). Whether it's a mono or multi-service package:
+ *   - Each service should have an alias format of <service_name>.example.dappnode.
+ *   - If it's the main service, also include example.dappnode.
+ *
+ * - For public packages ( example.public.dappnode.eth). Whether it's a mono or multi-service package:
+ *   - Each service should have an alias format of <service_name>.example.public.dappnode.
+ *   - If it's the main service, also include example.public.dappnode.
+ *
+ * @param dnpName dappnode dnp name
+ * @param serviceName container service name
+ * @param isMainOrMonoservice IF true returns the full network alias, otherwise the short network alias
+ * @param isExternal is the container external
+ * @param short if true, it will return the short version of the dnpName otherwise the full network alias
+ * @returns the root private network alias. It
  */
-export function shortUniqueDappnodeEns(dnpName: string): string {
-  for (const s of [".dnp.dappnode.eth", ".dappnode.eth", ".eth"])
-    if (dnpName.endsWith(s)) dnpName = dnpName.slice(0, -s.length);
-  return stripCharacters(dnpName);
-}
-
-export type ContainerNames = { serviceName: string; dnpName: string };
+export function determineNetworkAlias({
```

Comment:
I would call this function `buildNetworkAlias` or `generateNetworkAlias`, as determine is not very accurate and could look like it sets the value somewhere but returns void

Reply:
Done